packages/react-components/services/ReactNativeLogger.ts

```diff
@@ -58,6 +58,20 @@ export default class ReactNativeLogger {
     return errorMsg
   }
 
+  sanitizeError = (error: Error, valueToPurge?: string) => {
+    const message = this.getErrorMessage(error)
+
+    if (message.toLowerCase().includes('password') || message.toLowerCase().includes('key')) {
+      return new Error('Error message hidden for privacy')
+    }
+
+    if (valueToPurge) {
+      return new Error(message.replace(new RegExp(valueToPurge, 'g'), ''))
```

Comment:
Do we want to place something like `<purged>` instead of `''` here?

Reply:
Will do